sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -265,25 +277,41 @@ public ServiceCall getBlobPropertiesWithRestResponse(String containerName,
     /**
      * Reads the entire blob.
      *
+     * <p>
+     * This method will execute a raw HTTP GET in order to download a single blob to the destination.
+     * It is **STRONGLY** recommended that you use the {@link com.azure.android.storage.blob.transfer.TransferClient#download(String, String, String, File)}
+     * or {@link com.azure.android.storage.blob.transfer.TransferClient#download(String, String, String, Uri)} method instead - that method will
+     * manage the transfer in the face of changing network conditions, and is able to transfer multiple
+     * blocks in parallel.
+     *`
      * @param containerName The container name.
      * @param blobName      The blob name.
+     * @return the response containing blob bytes
      */
-    public ResponseBody download(String containerName,
-                                 String blobName) {
+    public ResponseBody rawDownload(String containerName,
```

Comment:
What is the reason behind renaming this to `rawDownload`? Would `download` be reserved for the `TransferClient`? Is the word `raw` something commonly used in the Java world for an operation that directly calls an endpoint or something like that?

Reply:
It was a preparation for the work of merging `TransferClient` APIs to `StorageClient`, similar to what iOS did, i.e. we want to discourage users from using raw GET APIs and encourage to use "managed" download. I'm not completely happy with this name though.

Reply:
Would it be a bad idea to keep `download` for both clients but to encourage people to use the managed operation both in the documentation and code samples? Any thoughts on this @bsiegel?

Reply:
Oh nevermind, looking at your other PR (#241) I piucked up on what you mean by merging both clients. I think having two different names is good, while I also feel there might be something better than the `raw` prefix.